00-project-documentation/communication/reports/turing-health-quartery-updates/Turing-final-report.md

```diff
@@ -0,0 +1,76 @@
+# The Alan Turing Institute final report
+
+12 September 2024
+
+This is the final report for AutSPACEs as a Turing funded project.
+It will contain main lessons and outputs to reflect the impact and facilitate the next phase.
+
+The report aims to be concise and point towards further resources, reports and outputs that have been shared and published along the way.
+
+**Contents**
+
+- [Project description & aims](#project-description-and-aims)
+- [Lessons learned](#-lessons-learned)
+- [Achievements](#achievements)
+- [Impact](#impact)
+- [Real world applications](#real-world-applications)
+- [Outputs](#outputs)
+
+## Project description and aims
+
+Short for **Aut**ism research into **S**ensory **P**rocessing for **A**ccessible **C**ommunity **E**nvironment**s**, [AutSPACEs is a co-created citizen science project](https://github.com/alan-turing-institute/AutSPACEs/) that was started in collaboration with the *[Autistica Foundation](https://www.autistica.org.uk/)*. The project was fully co-created with autistic participants that are involved in project design, management and implementation. The project was developing an online platform to collect qualitative experiences on sensory processing in autistic people, alongside recommendations & coping strategies to support autistic people, their friends/relatives, and policy makers. 
+
+The project had four goals: (1) Collect data to improve our understanding of sensory processing in people’s daily lives (2) Allow people to share their experiences and coping strategies to support others (3) Give non-autistic people a way to learn about the experiences of autistic people, allowing them to better understand the challenges and provide better support (4) Allow this data to be used by policy makers and creators of spaces and environments, to improve inclusion for autistic people.
+
+## Lessons learned
```

Comment:
Have we got any operational lessons learnt from the project management side? Apart from "build in project management time into grants"...

Reply:
Thinking in comment before writing into report:
- This approach ultimately made data management and ethics review processes much more straightforward to answer, nonetheless some of it can be a novelty to reviewers and clash with their usual best practives which made more assurances and explanations necessary
- The project changes in more unexpected ways after community input, which requires a more adaptative approach to managing it. With needs arising or changing along the project
- A project manager or similar figure is useful to convey to the community or collaborators, even researchers, hard lines or policies that must be followed and help them navigate those. This also helps researchers (I hope) focus on engagement and be receptive of their proposals
-to be continued

Reply:
I added into the report, left out bits and in particular the last item, it felt long enough and also focused on the most important bits

Reply:
We should capture these internally in the doc hub for PMU though!